### ISSUE TITLE
Loss and Perplexity

### ISSUE THREAD

**Issue by AmeenAli** [songlab-cal/tape#74]
Hi,
I run language modeling experiment with the tiny transformers model, and i get loss on the evaluation: 2.4 and preplexity 13.0. But these numbers are not reported in the paper , am I missing something?

**Comment by rmrao** [comment]
The experiments in the paper were performed using the old tensorflow repository (https://github.com/songlab-cal/tape-neurips2019). As a result it's possible there are discrepancies.

Without further information (did you match the hyperparameters for the model, for training, etc.) it's hard to tell what other issues there could be.

**Comment by AmeenAli** [comment]
@rmrao hi
I ran with the tiny transformers config with no more changes, got loss 2.5 and perplexity  13 which are better than the numbers reported in the paper..

**Comment by rmrao** [comment]
Again, like I said, the numbers reported in the paper are from the tensorflow respository. The tiny transformer config in this repository does not correspond to the tiny transformer from the paper.
